src/token/erc1155/interface.cairo

```diff
@@ -0,0 +1,180 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts for Cairo v0.8.1 (token/erc1155/interface.cairo)
+
+use starknet::ContractAddress;
+
+const IERC1155_ID: felt252 = 0xdef955e77a50cefb767c39f5e3bacb4d24f75e2de1d930ae214fcd6f7d42f3;
+const IERC1155_METADATA_ID: felt252 =
+    0x3d7b708e1a6bd1a69c8d4deedf7ad6adc6cda9cc81bd97c49dc1c82e172d1fc;
+const IERC1155_RECEIVER_ID: felt252 =
+    0x15e8665b5af20040c3af1670509df02eb916375cdf7d8cbaf7bd553a257515e;
+
+
+#[starknet::interface]
+trait IERC1155<TState> {
+    fn balance_of(self: @TState, account: ContractAddress, token_id: u256) -> u256;
+    fn balance_of_batch(
+        self: @TState, accounts: Span<ContractAddress>, token_ids: Span<u256>
+    ) -> Span<u256>;
+    fn safe_transfer_from(
+        ref self: TState,
+        from: ContractAddress,
+        to: ContractAddress,
+        token_id: u256,
+        value: u256,
+        data: Span<felt252>
+    );
+    fn safe_batch_transfer_from(
+        ref self: TState,
+        from: ContractAddress,
+        to: ContractAddress,
+        token_ids: Span<u256>,
+        values: Span<u256>,
+        data: Span<felt252>
+    );
+    fn is_approved_for_all(
+        self: @TState, owner: ContractAddress, operator: ContractAddress
+    ) -> bool;
+    fn set_approval_for_all(ref self: TState, operator: ContractAddress, approved: bool);
+}
+
+#[starknet::interface]
+trait IERC1155MetadataURI<TState> {
+    fn uri(self: @TState, token_id: u256) -> ByteArray;
+}
+
+#[starknet::interface]
+trait IERC1155Camel<TState> {
```

Comment:
any reason to have change `CamelOnly` to `Camel` ? I did to stay consistency with what was done for ERC-721 [her](https://github.com/OpenZeppelin/cairo-contracts/blob/861fc416f87addbe23a3b47f9d19ab27c10d5dc8/src/token/erc721/interface.cairo#L40)

Reply:
We use CamelOnly when there are functions outside the camel interface because they collide (like `name` or `symbol` that would be equal in both snake and camel). In this case, there are no colliding functions, and being the full interface we use just Camel.